near/contracts/ft/Cargo.toml

```diff
@@ -0,0 +1,26 @@
+[package]
+name = "ft"
```

Comment:
ft? fungible token? might help to spell out the package and directory name - also something about package names needing to be unique in a repo for cargo reasons, so may also want to consider throwing `near` and `wormhole` in there akin to the cosmwasm and solana stuff.

Reply:
added near-